client/src/components/TaskDurationBarChart/TaskDurationBarChart.module.css

```diff
@@ -0,0 +1,10 @@
+.scrollableTaskChart {
+  overflow: hidden;
+  overflow-x: scroll;
+}
+
```

Comment:
For the chart, it seems like the scrollbar is appearing (Although it doesn't allow you to scroll)
Perhaps we can remove this? (To make the look cleaner)
![image](https://user-images.githubusercontent.com/24628243/162016844-7fa32cdb-c829-42ef-851e-8f977e3b66c4.png)

Reply:
Fixed scrollbar so that it will not show when there is no overflow. It will show up when there is too many tasks to show on screen.